save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserProjectController.kt

```diff
@@ -95,9 +94,6 @@ class LnkUserProjectController(
         }
         .map { project ->
             lnkUserProjectService.getAllUsersAndRolesByProject(project)
-                .filter { (_, role) ->
-                    role != Role.NONE
```

Comment:
similar fix was missed by me in `com.saveourtool.save.backend.controllers.LnkUserOrganizationController#getAllUsersByOrganizationName`.
Can you fix in your PR?

Reply:
Yes sure. Fixed